R/badger.R

```diff
@@ -97,8 +99,8 @@ create_badges <- function(badges_data, output_file_name = NULL,
       badges_data[[col]] <- ""
   }
   cards <- paste((badges_data %>%
-                    rowwise() %>%
-                    mutate(cardcode = make_participant(first, second, role, extra)))$cardcode,
+                    dplyr::rowwise() %>%
```

Comment:
Here I see syntax conflict, if you use `@importFrom dplyr rowwise` then you should call only `rowwise()`. If you prefer to use `dplyr::rowwise()`, then roxygen import should be: `@import dplyr`

Reply:
Interesting. I've never realised that would cause an issue. AFAIK it's standard to namespace the function, then generate the `@importFrom` from that process. That's how `sinew` suggest it should work.

Reply:
I think this is the standard way Hadley suggests here: https://r-pkgs.org/namespace.html#imports so let's stick to that convention as it is widely adopted. Also, just noticed (from the source linked before) that just calling `@importFrom magrittr %>%` should do the job with the pipe. Could you test that? Maybe we don't need `utils-pipe.R` at all in the end.